### PR TITLE
Production push

### PR DESCRIPTION
**Pull request by paylakatel**

### CODE REVIEW

File: web-components/chart/chart.tsx [hunk 1]
```diff
@@ -12,21 +12,35 @@ export class CobChart {
...
   @Prop({ mutable: true })
   config: any;
+  chartDiv: any;
   chartID: string = '';
   minWidth: number = 0;
   view: any;
   dataset: any;
   selectField: string = '';
   selectOptions: any;
+  // True if using VegaLite, false if using Vega.
```
Comment (jessicamarcus) [inline web-components/chart/chart.tsx:22]: Great use of comments as documentation!

File: web-components/html/grouped-bar-chart.html [hunk 2]
```diff
@@ -1,102 +1,105 @@
...
+          "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
+          "boston": {
+            "chartID": "groupedBarSelect1",
+            "minWidth": 500,
+            "defaultSelection": "City Clerk"
           },
-        "mark": {
-          "type": "bar"
+          "title": {
+            "text": "Fake data by budget year",
```
Comment (jessicamarcus) [inline web-components/html/grouped-bar-chart.html:30]: Not sure where this code ends up, but wanted to confirm that this is the correct text for production.

**Review by jessicamarcus — APPROVED** [review]
I love how explicit you are with documentation comments! 😄

File: web-components/html/grouped-bar-chart.html (hunk 2 above)
Comment (paylakatel) [reply web-components/html/grouped-bar-chart.html:30]: Yup - its an example graph and I wanted it to be clear that its not real data.

**Pull request merged**